frontend/app/src/components/buttons/tabs-buttons.tsx

```diff
@@ -21,15 +21,13 @@ export const TabsButtons = (props: TabsProps) => {
 
   return (
     <div className="bg-custom-white flex items-center">
-      <div className="isolate inline-flex rounded-md shadow-sm border border-gray-300 m-4 overflow-hidden">
+      <div className="isolate inline-flex rounded-md shadow-sm border border-gray-300 overflow-hidden">
```

Comment:
overflow-hidden prevent focus from being visible. Since it already here we can fix that later